mantidproject/mslice
# Implement Cut Tab

Description of work.

**To test:**
Select an MD workspace in the worskpace manager,  fill in the required parameters for the cut and click plot/ plot_over should bring up plots

clicking on save cut to workspace will save cuts to workspaces and show them in workspace manager. Selecting any one of these cuts will disable the cut tab except plot/plot over and intensity range and the show the cuts parameters.

Specifying a width will divide the integration range into blocks of specified width and will then cause multiple cuts to be saved/plotted

Fixes #27 
Fixes #45 
Fixes #44 
Fixes #47 
Fixes #46
